asm/f64/asm_test.go

```diff
@@ -65,3 +68,48 @@ func nonStridedWrite(x []float64, inc int) bool {
 	}
 	return false
 }
+
+type incSet struct {
+	x, y int
+}
+
+func genInc(inc []int) []incSet {
```

Comment:
Please add a doc comment to these two helpers and their associated types. I would also try to think of better names so that the tests are more fluently read. I'd recommend `newIncSet` and `newIncToSet` for the functions.

For both, you may want to use the variadic form for `inc`.